spec/index.html

```diff
@@ -788,106 +788,79 @@ <h3>Quoted Graphs</h3>
-        In the forward direction, a path describes a relationship from a specific
-        <a>N3 triple element</a> (such as an <a>IRI</a>) to a <a>blank node</a>.
-        In the reverse direction, it describes the relationship
-        from a <a>blank node</a> to some <a>N3 triple element</a>.
-        Successive indicator/<a href="#grammar-production-pathItem">pathItem</a>
-        pairs describe subsequent relationships with the trailing
-        <a href="#grammar-production-pathItem">pathItem</a> describing
-        the relationship <a data-cite="RDF11-CONCEPTS#dfn-predicate">predicate</a>.
-      </p>
-
-      <p class="note">In SPARQL, a property path is used in the
-        <a data-cite="RDF11-CONCEPTS#dfn-predicate">predicate</a>
-        position to describe a string of relationships between
-        a <a data-cite="RDF11-CONCEPTS#dfn-subject">subject</a>
-        and an <a data-cite="RDF11-CONCEPTS#dfn-object">object</a> resource.
-        Although unrestricted in N3, a
-        <a>resource path</a> is most often used in the
-        <a data-cite="RDF11-CONCEPTS#dfn-subject">subject</a> or
-        <a data-cite="RDF11-CONCEPTS#dfn-object">object</a> position.
-      </p>
+      <p>Similar to <a data-cite="SPARQL11-QUERY#propertypaths">SPARQL property paths</a> [[SPARQL11-QUERY]], 
+        N3 <dfn>resource paths</dfn> concisely express paths between resources, 
+        in case intermediary resources on the path are not relevant.</p>
 
-      <p>Each path segment can be thought of as a shorthand for
-        an existing relationship.
-        For example, for a
-        <a data-cite="RDF11-CONCEPTS#dfn-predicate">predicate</a> |p|
-        and <a href="#grammar-production-pathItem">pathItem</a> or <a>path</a> |x|,
-        <code><var>x</var>!<var>p</var></code> is equivalent to
-        <code>[ is <var>p</var> of <var>x</var> ]</code>.
-        Similarly, <code><var>x</var>^<var>p</var></code> is equivalent to
-        <code>[ <var>p</var> <var>x</var> ]</code>.
-        Thus, the result is always a <a>blank node</a>.
-      </p>
-
-      <p>The expansion of a <a>path</a> is equivalent to unwinding
-        these relationships.</p>
-
-      <aside class="example" title="Forward path chain">
-        <p>The following example of a forward path describes
-          a relationship between Joe's mother's office's zipcode:</p>
+        <p>A <a>resource path</a> starts from a subject resource, followed by one or more predicates;
+          each predicate is separated by a <dfn>directional indicator</dfn> to follow the predicate either forward (`!`) or in reverse (`^`).
+        For example, the following example describes the city of Joe's mother's office's address:</p>
 
-        <pre data-transform="updateExample">
+        <pre class="example nohighlight" data-transform="updateExample">
             <!--
-            @base <http://example.org/#> .
+            @prefix : <http://example.org/#> .
```

Comment:
```suggestion
@prefix : <http://example.org/#> .
```

Reply:
Indenting pre blocks shouldn't be necessary, as ReSpec does this automatically now.

Reply:
Rather, left-justifying.

Reply:
> ReSpec does this automatically now

Ah, OK. They do keep improving!